docs/source/models/concept_models/vivarium_nutrition_optimization/kids/concept_model.rst

```diff
@@ -512,9 +512,17 @@ pregnancy sim, will be national. Similarly, the PAFs for LBWSG will also
 be national. All other GBD data will be at the subnational level.
 
 Artifacts will be made for all subnational 
-geographies. We will also regenerate data for all custom made datasets, 
-such as wasting transitions, PAFs, and CGF correlation, at the subnational 
-level. 
+geographies. We will also regenerate most data for custom made datasets, 
+such as wasting transitions and PAFs.
+
+CGF correlation data will continue to be at the national level. This is 
+following a analysis of DHS data which showed little subnational variation 
+and small sample sizes for many locations. Correlation coefficients generally 
+only varied by a maximum of 0.1 points (for example, between 0.6 and 0.7). 
+Furthermore, no geographic patterns such as north/south or urban/rural were 
```

Comment:
Should we make a few maps before we move on?  @pletale has just been working on mapping code, they might be able to do it or tell you how to do.  People love maps!

Reply:
I'll add this as an optional add-on later! If we want to validate this for a paper or something we can definitely add in a few maps.